test/language/expressions/object/method-definition/generator-super-prop-body.js

```diff
@@ -10,8 +10,18 @@ author: Sam Mikes
 description: GeneratorMethod body uses SuperProperty (allowed)
 ---*/
 
-var obj = {
-    *foo(a) {
-        yield super.x;
+var value = {};
+var obj;
+
+try {
+  Object.prototype.Test262Attr = value;
+  obj = {
+    *foo() {
+      return super.Test262Attr;
```

Comment:
could you return super.toString and ensure that it is same value as Object.prototype.toString?